pcf/test/particle/aws/testdata.json

```diff
@@ -0,0 +1,167 @@
+{
+  "hosted-zone-apply": [
+    {
+      "pcf_name": "pcf_hosted_zone",
+      "flavor": "route53_hosted_zone",
+      "aws_resource": {
+        "Name": "www.hoooooos.com.",
+        "custom_config": {
+          "Tags": [
+            {
+              "Key": "Owner",
+              "Value": "Hoo"
+            },
+            {
+              "Key": "UID",
+              "Value": "abc123"
+            }
+          ]
+        },
+        "VPC": {
+          "VPCRegion": "us-east-1",
+          "VPCId": "vpc-12345"
+        },
+        "CallerReference": "sdfa6df5a4sdf5asd7f9asd6fa5s",
+        "HostedZoneConfig": {
+          "Comment": "hoo",
+          "PrivateZone": true
+        }
+      }
+    },
+    {
+      "pcf_name": "pcf_hosted_zone",
```

Comment:
Can we change this so only the updated fields are here. Then you can just use the update util in the test function to get the full updated definition. This should reduce the file size a lot and make it much easier to read.

Reply:
made all the changes and pushed, but was wondering

what if a tag is removed? using update_dict still includes that removed tag
It's not a big deal, so I just pushed anyway, but wanted to make sure it wouldn't be an issue